lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +264,61 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._mystem_analyzer = Mystem()
+        mapping_path = Path(__file__).parent / 'data' / 'mystem_tags_mapping.json'
+        self._converter = MystemTagConverter(mapping_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = []
+        word_regex = re.compile(r'\w+|[.]')
+
+        for sentence_idx, sentence in enumerate(split_by_sentence(text)):
+            conllu_tokens = []
+            sentence_counter = 0
+            mystem_analysis = self._mystem_analyzer.analyze(sentence)
+
+            for token in mystem_analysis:
+                if not word_regex.match(token['text']):
+                    continue
+                sentence_counter += 1
+                token['text'] = token['text'].replace(' ', '')
```

Comment:
what is the real case, when this substitution is used? can you provide an example?

Reply:
tests for both labs should be running

Reply:
you have general tests errors that do not allow you to run lab 6 tests

Reply:
see code style check log for more details

Reply:
locally, all tests for a score of 8 for lab 6 are passed, but there are no such tests in PR(

Reply:
fixed